test_task_journey.py

```diff
@@ -0,0 +1,8 @@
+import os
+import sys
+from firefox_driver import driver
+import test_app
```

Comment:
This needs to be
`from test_app import test_app` to specifically import the fixture

Otherwise I get an error